CorsixTH/Lua/languages/english.lua

```diff
@@ -911,7 +911,11 @@ map_editor_window = {
     heliport_3 = "Heliport 3",
     heliport_4 = "Heliport 4",
     paste = "Paste area",
-  }
+  },
+  checks = {
+    spawnpoints = "Warning: Spawn points could not be generated for this map.",
```

Comment:
Is a "spawn point" something that a user is supposed to know?

maybe "Warning: New patients cannot enter the map." ? (better connection to patients, but not entirely happy with the sentence).

Reply:
> Warning: There are no tiles from the road page on the edges of the map. Patients need these to enter the map.

That directly gives them the remedy and reason.